src/renault_api/kamereon/models.py

```diff
@@ -73,6 +78,11 @@
         "support-endpoint-location": False,
         "support-endpoint-lock-status": False,
     },
+    "AVN": {  # MEGANE E-TECH
+        "reports-in-watts": True,
+        "reports-charge-session-durations-in-minutes": True,
+        "support-endpoint-lock-status": False,
+    },
```

Comment:
I don't think these lines are correct.
```suggestion
```

Reply:
I thought it was necessary since the car seems to report differently from the others. But happy to remove this @epenet